website/docs/d/outputs.html.markdown

```diff
@@ -9,10 +9,8 @@ description: |-
 This data source is used to retrieve the state outputs for a given workspace.
 It enables output values in one Terraform configuration to be used in another.
 
-The `values` attribute has been defined as sensitive because the workspace
-outputs may contain sensitive values and the provider protocol does not support
-marking values as [sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) after this information is known. You can access
-the subset of non-sensitive outputs using the `nonsensitive_values` attribute.
+~> **NOTE:** The `values` attribute is preemptively marked sensitive and is only populated after a run completes on the associated workspace. To access non-sensitive values, use the `nonsensitive_values` attribute, which is a subset of outputs
```

Comment:
I think we can merge the first and last segments of the sentence since they seem to say the same thing, that is `To access non-sensitive values...` and `...that are known to not be sensitive`. What do you think about something in the lines of, 
```
Use the `nonsensitive_values` attribute to access a subset of the outputs
that are known to be non-sensitive

```